instrument/NOMAD_Parameters.xml

```diff
@@ -1,14 +1,125 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <parameter-file instrument = "NOMAD" valid-from = "2011-08-13T00:00:00">
 
-<component-link name = "NOMAD">
+    <component-link name = "NOMAD">
 
-<parameter name="TS_mapping_file" type="string">
-    <value val="NOM_TS_2010_12_01.dat"/>
-</parameter>
+        <parameter name="TS_mapping_file" type="string">
+            <value val="NOM_TS_2010_12_01.dat"/>
+        </parameter>
 
+        <parameter name="IkedaCarpenterPV:Kappa" type="fitting">
+            <formula eq="0.1"/>
```

Comment:
I think this need to `result-unit` to work, so can you make this `<formula eq="0.1" result-unit="TOF"/>`.